R/xcmsSet.R

```diff
@@ -1036,11 +1032,11 @@ setMethod("retcor.peakgroups", "xcmsSet", function(object, missing = 1, extra =
 
         screen(2)
         par(mar = c(5.1, 4.1, 0, 2), yaxt = "n")
-        allden <- density(peakmat[,"rt"], bw = diff(rtrange)/200, from = rtrange[1], to = rtrange[2])[c("x","y")]
+        allden <- density(peakmat[,"rt"], bw = diff(rtrange)/200, from = rtrange[1], to = rtrange[2], na.rm=TRUE)[c("x","y")]
```

Comment:
hi Steffen,
concerning the first note, I'll address that. To the second, I inserted that code when I was looking for the reason I got the error, which was however related to the col parameter. I will remove that again and submit the code again.